minutes/2019-11-12.md

```diff
@@ -0,0 +1,65 @@
+# Jakarta EE Platform Call #2
+
+General
+Date: 2019-11-12
+
+Present:
+
+* Kevin Sutter
+* Bill Shannon
+* Ivar Grimstad
+* Emily Jiang
+* Tanja Obradovic
+* Martin Stefanko
+* Ken Finnigan
+* Edwin Amoakwa
+* Carlos Andres De La Rosa
+* Steve Millidge
+* Rudy De Busscher
+* Alessio Soldano
+
+## Agenda
+
+* Organize ourselves
+
+Kevin (IBM) and Steve (Payara) volunteered to take on the Delivery Owner and Co-pilot role for Jakarta EE 9. Final confirmation pending.
+
+Keep the current meeting time: 11:00 AM EST.
```

Comment:
Add "every Tuesday, except when it conflicts with the PMC meeting and then the PMC meeting takes precedence." Or, something to that effect.

Reply:
"every Tuesday, except when it conflicts with the PMC meeting (currently every second Tuesday of the month) and then the PMC meeting takes precedence"

A link to the PMC meeting schedule would be nice.

It would also be nice if the meeting chair (Ivar?) would commit to sending a reminder for the meeting along with the proposed agenda at least 24 hours in advance of the meeting.

Reply:
done